curve25519-dalek/src/constants.rs

```diff
@@ -86,7 +69,10 @@ pub const RISTRETTO_BASEPOINT_POINT: RistrettoPoint = RistrettoPoint(ED25519_BAS
 /// $$
 /// \ell = 2^\{252\} + 27742317777372353535851937790883648493.
 /// $$
-pub const BASEPOINT_ORDER: Scalar = Scalar {
+#[deprecated(since = "5.0.0", note = "Should not have been in public API")]
```

Comment:
Shouldn't this be whatever the next release is, e.g.

```suggestion
#[deprecated(since = "4.1.1", note = "Should not have been in public API")]
```

Reply:
True, thanks